### DISCUSSION TITLE
Discussion labeling

### DISCUSSION THREAD

**Discussion by balanza** [pagopa/technology-radar#36] [Show and tell]
I spent 10 minutes creating some labels to better categorize discussions:

* **chore**: tech stuff about the code of the radar (site generator, CI/CD, hosting, templates, etc)
  * _"should we add a preview for every PR?"_
* **choose the ring**: a discussion about which ring an item fits best
  * _"is X adopt or trial?"_
* **how to contribute**: a discussion about how someone can contribute to the radar (content, format, templates, process, roles, etc)
  * _"who can merge a PR?"_
* **taxonomy**: to review quadrants, rings and tags
  * _"should we add a tag for CSS?"_
  
The idea is to have a glimpse on what the discussion is about (so we can be more focused and effective).

Feedback welcolme 👋

**Comment by gunzip** [comment]
Regarding "choose the ring," it is typically driven on the pull request itself rather than on this forum.

**Reply by gunzip** [reply]
Even "taxonomy" discussions should be triggered by a proposal (= PRs) imho. What I'm expeting from this forum falls mostly in the category "how to contribute".
